app/models/publications/itt_monthly_report_generator.rb

```diff
@@ -16,12 +19,45 @@ def to_h
           title: I18n.t('publications.itt_monthly_report_generator.candidate_headline_statistics.title'),
           data: candidate_headline_statistics,
         },
+        candidate_age_group: {
+          title: I18n.t('publications.itt_monthly_report_generator.age_group.title'),
+          data: candidate_age_group,
+        },
+        candidate_sex: {
+          title: I18n.t('publications.itt_monthly_report_generator.sex.title'),
+          data: candidate_sex,
+        },
+        candidate_area: {
+          title: I18n.t('publications.itt_monthly_report_generator.area.title'),
+          data: candidate_area,
+        },
+        candidate_phase: {
+          title: I18n.t('publications.itt_monthly_report_generator.phase.title'),
+          data: candidate_phase,
+        },
       }
     end
 
+    def explain
+      {
+        candidate_headline_statistics_query: candidate_headline_statistics_query(cycle_week:),
+        age_group_query: age_group_query(cycle_week:),
+        sex_query: sex_query(cycle_week:),
+        area_query: area_query(cycle_week:),
+        phase_query: phase_query(cycle_week:),
+      }.each do |key, value|
+        # rubocop:disable Rails/Output
+        puts "========= #{key.to_s.humanize} =========="
+        puts value
+        puts '=' * 40
+        # rubocop:enable Rails/Output
+      end; nil
+    end
```

Comment:
I decided to add the explain method to maintain a mental sanity around each query we do on BigQuery and each section on the report. Firstly I had the explain method on `DfE::Bigquery::ApplicationMetrics` but then I moved to here because conceptually it makes sense to `show all the queries from this report` to be on the report itself.

Reply:
Done! Renamed to describe